### PR TITLE
Add keys for Java 11 related dependency update

### PR DESCRIPTION
**Pull request by krystofNovotny**
I have added new key for every dependency that changed signature and added a few new keys.

### CODE REVIEW

File: src/main/resources/trustedkeys.properties [hunk 1]
```diff
@@ -16,23 +16,28 @@ cglib:cglib-nodep=0x52E6585E0102B84D
 ch.qos.logback=0x2C7B12F2A511E325
 com.amazonaws=0x03BD3C33F16AB41B
 com.barchart.udt:barchart-udt-bundle=0x80B2E0AA2E117339
-com.beust:jcommander=0x73836387848241D7, 0xCC54B93C018511A4, 0x930C5B1EA41B1AA1, 0x5AD66315FC018797
-com.fasterxml.*=0x9CD8549ACF9BD0CE, 0x1947DCDB647536A9, 0xC9FBAA83A8753994
+com.beust:jcommander=0x73836387848241D7, 0xCC54B93C018511A4, 0x930C5B1EA41B1AA1, 0x5AD66315FC018797, 0x22E44AC0622B91C3
+com.fasterxml.*=0x9CD8549ACF9BD0CE, 0x1947DCDB647536A9, 0xC9FBAA83A8753994, 0x8D7F1BEC1E2ECAE7
```
Comment (pavelhoral) [inline src/main/resources/trustedkeys.properties:20]: Was not able to find any information on this key.

File: src/main/resources/trustedkeys.properties [hunk 2]
```diff
@@ -16,23 +16,28 @@ cglib:cglib-nodep=0x52E6585E0102B84D
...
 com.googlecode.json-simple=0x717134BDA955E8F3
+com.google.guava:failureaccess=0x7457CA33C3CE9E15
 com.google.guava:guava=0x00ECB0B904931A76, 0x806E7D417A281864, 0xABE9F3126BB741C1
 com.google.guava:guava-jdk5:13.0=0x021E5B388199BCE2
+com.google.guava:listenablefuture=0xF6D4A1D411E9D1AE
 com.google.http-client=0xD01DBF93D06A19ED
 com.google.inject.extensions=0x52E6585E0102B84D
 com.google.inject=0x52E6585E0102B84D, 0xABE9F3126BB741C1
+com.google.inject:guice=0xF0DF21D1D0A3C384
+com.google.j2objc:j2objc-annotations=0x29579F18FA8FD93B
```
Comment (pavelhoral) [inline src/main/resources/trustedkeys.properties:40]: Why we need this dependency?

File: src/main/resources/trustedkeys.properties (hunk 1 above)
Comment (pavelhoral) [reply src/main/resources/trustedkeys.properties:20]: https://www.mail-archive.com/jackson-dev@googlegroups.com/msg00370.html

File: src/main/resources/trustedkeys.properties (hunk 2 above)
Comment (pavelhoral) [reply src/main/resources/trustedkeys.properties:40]: This dependency with this exact key is used in gradle's official dependency verification guide :D 

https://docs.gradle.org/current/userguide/dependency_verification.html

Comment (krystofNovotny) [reply src/main/resources/trustedkeys.properties:40]: It's a part of `testng 7.3.0 -> guice 4.2.2.- no_app -> guava:25.1-android -> j2objc-annotations` dependency chain

**Pull request merged**